shared/cfg/guest-os/Linux/RHEL.cfg

```diff
@@ -5,7 +5,7 @@
         modprobe_module = acpiphp
     block_hotplug:
         modprobe_module = acpiphp
-    unattended_install, check_block_size.4096_512, check_block_size.512_512, svirt_install, with_installation:
+    unattended_install, check_block_size, svirt_install, with_installation:
```

Comment:
From you commit, I guess you just wanna enable it on ovmf, but you patch is enable all the machine type, right?

Reply:
Yes, i think this is controlled in check_block_size.cfg is better instead of specific case. it is hard to maintain.